src/window.py

```diff
@@ -470,7 +469,7 @@ def _on_rename_updated(self, cell_name, treepath, new_name, data=None):
             row = self.liststore.get_iter(_treepath)
             self.liststore.set_value(row, 2, new_path)
             self.liststore.set_value(row, 1, new_name)
-        except:
+        except FileExistsError:
```

Comment:
It's not pretty, but we should rather just use Exception here as there are other possible errors here.

Reply:
Hmm ok, but the except clause follows logs for file exists warnings only. Anyway, I removed them now